src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -234,7 +236,43 @@ object TelemetryTypes extends Enumeration {
   val SPARK_PUSHDOWN_FAIL: Value = Value("spark_pushdown_fail")
 }
 
-object TelemetryConstValues {
+// All telemetry message field names have to be defined here
+// Important:
+//   1. These fields name have been used in released spark connector.
+//      change this may cause telemetry message query to be difficult.
+//   2. Make sure to review whether existing field names to be qualified
+//      before adding new name.
+private[snowflake] object TelemetryFieldNames {
+  val SPARK_CONNECTOR_VERSION = "spark_connector_version"
+  val SPARK_VERSION = "spark_version"
+  val APPLICATION_NAME = "application_name"
+  val SCALA_VERSION = "scala_version"
+  val JAVA_VERSION = "java_version"
+  val JDBC_VERSION = "jdbc_version"
+  val CERTIFIED_JDBC_VERSION = "certified_jdbc_version"
+  val SFURL = "sfurl"
+  val OPERATION = "operation"
+  val QUERY = "query"
+  val QUERY_ID = "query_id"
+  val QUERY_STATUS = "query_status"
```

Comment:
To reviewer:
There are 2 actual fields name changes:
1. For QUERY_STATUS, "status" -> "query_status", this is only used in new telemetry message spark_query_status,  I believe it is no problem.
2. For QUERY_ID with released OOB message, "queryid" -> "query_id". This doesn't cause any change for customer. When we need to to query with filter on QUERY_ID in snowhouse, we can query both old and new values. So I think it should be fine.